app/build.gradle

```
@@ -59,7 +59,7 @@ flyway {
     def port = System.getenv("HOSPEBEM_DB_PORT") ?: 5432
     def name = System.getenv("HOSPEBEM_DB_NAME") ?: 'hospebem'
 
-    url = "jdbc:postgresql://$host:$port/$name"
+    url = System.getenv("SPRING_DATASOURCE_URL") ?: 'jdbc:postgres://gogwdmoefhpmpn:414a8fa4deb0196f2c96a03918f3da0e8d0e35ef63e91d9271236fd1de61087a@ec2-184-73-222-192.compute-1.amazonaws.com:5432/d4blq8a0ehm41q'
```

NÃOOOOOOOOOOOOOOOOOOOOOOOOOOOOO

---

Pessoal, vocês não podem commitar dados de acesso ao banco de dados. Essa URL contém usuário e senha do banco de dados de vocês. Isso 

# Não pode ser público

Para isso estamos usando as variáveis de ambiente.

---

**Troquem as credenciais do postgres** e usem variáveis de ambiente.

---

O que acontece aqui é o uso do [Elvis Operator](https://en.wikipedia.org/wiki/Elvis_operator).

Basicamente, isso significa:

Se a variável `SPRING_DATASOURCE_URL` não estiver configurada na máquina em que o projeto está rodando, use esse valor default.

O valor que o Heroku configura (que está commitado aqui) __NUNCA__ deve aparecer no código, ele deve sempre ser configurado através de variáveis, pois é uma informação sensível.

```suggestion
    url = System.getenv("SPRING_DATASOURCE_URL") ?: 'jdbc:postgres://postgres:hospebem@localhost:5432/hospebem'
```